lib/logstash/inputs/kafka.rb

```diff
@@ -128,69 +128,78 @@ def register
     elsif topic_or_filter.count > 1
       raise LogStash::ConfigurationError, 'Invalid combination of topic_id, white_list or black_list. Use only one.'
     end
-    @kafka_client_queue = SizedQueue.new(@queue_size)
-    @consumer_group = create_consumer_group(options)
-    @logger.info('Registering kafka', :group_id => @group_id, :topic_id => @topic_id, :zk_connect => @zk_connect)
+    @consumer = create_consumer(options)
   end # def register
 
   public
   def run(logstash_queue)
-    # noinspection JRubyStringImportInspection
-    java_import 'kafka.common.ConsumerRebalanceFailedException'
-    @logger.info('Running kafka', :group_id => @group_id, :topic_id => @topic_id, :zk_connect => @zk_connect)
     begin
-      @consumer_group.run(@consumer_threads,@kafka_client_queue)
-
-      while !stop?
-        event = @kafka_client_queue.pop
-        if event == KAFKA_SHUTDOWN_EVENT
```

Comment:
since `KAFKA_SHUTDOWN_EVENT` is no longer being used. we can remove mentions of that from the file.